Sources/BraintreeCoreSwift/BTAPIHTTP.swift

```diff
@@ -0,0 +1,23 @@
+import Foundation
+
+// TODO: Make internal once rest of core is in Swift
+@objcMembers public class BTAPIHTTP: BTHTTP {
+
+    let accessToken: String?
+
+    @objc(initWithBaseURL:accessToken:)
+    public init(url: URL, accessToken: String? = "") {
```

Comment:
Is [this logic](https://github.com/braintree/braintree_ios/blob/91cfc9567622099af7764b512f1104af9c15a9a5/Sources/BraintreeCore/BTAPIHTTP.m#L17-L25) from the ObjC implementation preserved?

Reply:
Yep, this is all done in `BTHTTP` now since it is the same across all of the classes that subclass `BTHTTP`